src/sidebar/components/HypothesisApp.tsx

```diff
@@ -64,6 +71,22 @@ function HypothesisApp({
     }
   }, [isSidebar, profile, settings, store]);
 
+  // As soon as the user interacts with the page creating annotations, if the
+  // help panel was auto-displayed, schedule a task that will close it after 5
+  // minutes
+  useEffect(() => {
+    if (
+      shouldAutoDisplayTutorial(isSidebar, profile, settings) &&
+      currentUserHasAnnotations &&
+      !timer.current
+    ) {
+      timer.current = setTimeout(() => {
+        store.closeSidebarPanel('help');
+        session.dismissSidebarTutorial();
+      }, 5 /** 60*/ * 1000);
+    }
+  }, [currentUserHasAnnotations, isSidebar, profile, session, settings, store]);
```

Comment:
I'm thinking maybe this logic should be moved to the `HelpPanel` itself.

In there we are also handling that the `show_sidebar_tutorial` profile preference is disabled if the panel is manually closed.

When that happens, we should also clear the timeout if it is still in progress.